_component_design/chips.md

```diff
@@ -17,27 +17,87 @@ helix-ui-javascript: false
 
 <section class="static-section" markdown="1">
 
-## Introduction
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+## When to use
+
+Use chips to graphically represent a unit of information. For example, use chips to represent a user or device.
+
+## Best practices
+
+Consider the following best practices when constructing a chip:
+
+- Avoid using chips in multiple locations on a page.
+- Label the chip so that the user can identify the information represented by the chip.
+
+{% endcolumn %}
+
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+
+## Specifications
```

Comment:
composition graphic should be centered and use the dots at the end of the line. 
also, add the text identifier into the composition image so the dot at the end of the redline is on the text